doc/api.yml

```diff
@@ -1245,6 +1354,19 @@ components:
           type: string
         email:
           type: string
+    Pointing:
+      properties:
+        id:
+          type: string
+        pointing_datetime:
```

Comment:
Are you sure we always used snake case ?

Reply:
yes all attribute or params are in snake case